torch/linalg/__init__.py

```diff
@@ -145,7 +145,46 @@
 det = _add_docstr(_linalg.linalg_det, r"""
 linalg.det(input) -> Tensor
 
-Alias of :func:`torch.det`.
+Computes the determinant of a square matrix or batch of square matrices.
```

Comment:
Tweak this language and give the HTML a last sanity check. Overall this looks great!